From 0bfed72dcae4b3054e62dcfbd9a828fad8fb2050 Mon Sep 17 00:00:00 2001
From: Xihua Chen <xihua.chen@intel.com>
Date: Thu, 14 Mar 2019 15:52:27 +0800
Subject: [PATCH 1085/1214] support reboot with "dm-verity device corrupted"
 command

If met dm-verity device corrupted case, it will reboot with
this command, but this command is not accepted in abl reboot
flow, add this can let the device enter into recovery

Change-Id: I0f8f0ffac61dde6af5a3340e65b289837eb3825b
Tracked-On: OAM-77043
Signed-off-by: Xihua Chen <xihua.chen@intel.com>
---
 drivers/staging/android/abl/ablbc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/staging/android/abl/ablbc.c b/drivers/staging/android/abl/ablbc.c
index cdcf726..78bfd02 100644
--- a/drivers/staging/android/abl/ablbc.c
+++ b/drivers/staging/android/abl/ablbc.c
@@ -100,6 +100,7 @@ static const struct name2id NAME2ID[] = {
 	{ "fastboot",	0x01 },
 	{ "elk",	0x02 },
 	{ "recovery",	0x03 },
+	{ "dm-verity device corrupted",	0x03 },
 	{ "crashmode",	0x04 },
 	{ "dnx",	0x05 },
 	{ "cli",	0x10 },
-- 
2.7.4

